src/main/java/de/tum/in/www1/artemis/repository/SubmissionRepository.java

```diff
@@ -117,6 +117,9 @@
     long countByExerciseIdSubmittedAfterDueDate(@Param("exerciseId") long exerciseId);
 
     /**
+     * returns submissions for a exercise. Returns only a submissoin that has a result with a matching assessor. Since the results list may also contain
```

Comment:
```suggestion
     * Returns submissions for a exercise. Returns only a submission that has a result with a matching assessor. Since the results list may also contain
```